lib/chain-utils/src/lib.rs

```diff
@@ -4,7 +4,7 @@
 pub mod berachain;
 pub mod cosmos;
 pub mod ethereum;
-pub mod scroll;
+// pub mod scroll;
```

Comment:
delete?

Reply:
keeping for the same reason as i kept the arbitrum one https://github.com/unionlabs/union/pull/3130#discussion_r1808074718